aries-site/src/pages/templates/toast-notifications.mdx

```diff
@@ -132,6 +133,13 @@ Follow these placement guidelines for Toast Notifications:
 - Reminder that toasts should not be used to convey a batch of information as this would quickly hinder their visibility and cause notification fatigue.
 - Consider incorporating a Notification Center where a user can access full information about tasks and asynchronous events/alerts.
 
+
```

Comment:
```suggestion
### Top-right Aligned Toast
```

Reply:
Just a suggestion to call out a little more that this example is for top-right aligned toast.

Reply:
I like that I was actually thinking about adding this as well when I put it up